wasmplugin/plugin.go

```diff
@@ -100,8 +95,34 @@ func (ctx *corazaPlugin) OnPluginStart(pluginConfigurationSize int) types.OnPlug
 		return types.OnPluginStartStatusFailed
 	}
 
+	directivesAuthoritiesMap := map[string][]string{}
+	for authority, directivesName := range config.perAuthorityDirectives {
+		if _, ok := directivesAuthoritiesMap[directivesName]; !ok {
```

Comment:
This isn't needed since nil slice can be appended to

Reply:
Done.